packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -28,6 +28,106 @@ export enum ReducerID {
   allIsNull = 'allIsNull',
   allValues = 'allValues',
   uniqueValues = 'uniqueValues',
+  p1 = 'p1',
```

Comment:
Ooof, @timlevett you're right, this is rough! 😊 We could remove the p values and replace it with something like this?

```
function generatePercentiles() {
  const percentiles: { [key: string]: string } = {};
  for (let i = 1; i <= 99; i++) {
    percentiles[`p${i}`] = `p${i}`;
  }
  return percentiles;
}

export enum ReducerID {
  ...generatePercentiles(),
}
```
?

Reply:
TS, c'mon! Well, we tried! ;)